internal/cli/cli.go

```diff
@@ -83,6 +87,24 @@ func (opts *deleteOpts) DeleteFromProject(d DeleterFromProject, projectID string
 	return nil
 }
 
+// DeleterFromProjectAuthDB deletes a resource from a project, it expects a callback
+// that should perform the deletion from the store.
+func (opts *deleteOpts) DeleterFromProjectAuthDB(d DeleterFromProjectAuthDB, projectID string) error {
+	if !opts.confirm {
+		fmt.Println(opts.failMessage)
+		return nil
+	}
+	err := d(opts.authDB, projectID, opts.entry)
```

Comment:
like I mentioned before we should pass `authDB` as a parameter and not as a member of `opts`